hermes/hermes.go

```diff
@@ -56,40 +57,12 @@ type StateJournal struct {
 	filenames map[int]string `json:"filenames"`
 }
 
-// Init initialises the map in the StateJournal so that entries can be added to it.
 func (sj *StateJournal) Init() {
-	sj.filenames = make(map[int]string) // initialise filenames to a map
+	sj.filenames = make(map[int]string)
 }
 
 // Hermes is the main Hermes prober that will startup Hermes and initiate monitoring targets.
```

Comment:
Nit: This comment is hard to parse, reminds me of "Buffalo buffalo Buffalo buffalo buffalo buffalo Buffalo buffalo".
https://en.wikipedia.org/wiki/Buffalo_buffalo_Buffalo_buffalo_buffalo_buffalo_Buffalo_buffalo

Reply:
That is an interesting read. I've rephrased this comment to be more descriptive and less repetitive.